test/logger.go

```diff
@@ -95,7 +95,21 @@ func collectManagementClusterLogs(bootstrapClusterProxy *e2e.AzureClusterProxy,
 
 	framework.DumpAllResources(context.TODO(), framework.DumpAllResourcesInput{
 		Lister:         bootstrapClusterProxy.GetClient(),
-		KubeConfigPath: bootstrapClusterProxy.GetKubeconfigPath(),
+		KubeConfigPath:       bootstrapClusterProxy.GetKubeconfigPath(),
+		ClusterctlConfigPath: getClusterctlConfigPath(),
+		Namespace:            *namespace,
+		LogPath:              workLoadClusterLogPath,
+	})
+}
+
+func getClusterctlConfigPath() string {
+	config := os.Getenv("CLUSTERCTL_CONFIG")
+	if config == "" {
+		return path.Join(getArtifactsFolder(), "repository", "clusterctl-config.yaml")
+	}
+
+	return config
+}
 		Namespace:      *namespace,
```

Comment:
Oh, I guess I didn't create the diff in such a way that it can merge directly, so now there are trailing lines 113-116 that should be removed. Sorry about that!

Reply:
No worries. I have fixed it now. Thanks for the code snippet.

Reply:
@mboersma it is now giving errors like

"failed to initialize the configuration reader: failed to check if clusterctl config file exists: stat /logs/artifacts/repository/clusterctl-config.yaml: no such file or directory"